lib/widget/input/dropdown.dart

```diff
@@ -73,6 +73,7 @@ abstract class SelectOne extends StatefulWidget
   @override
   Map<String, Function> methods() {
     return {
+      'getValue': () => getValue(),
```

Comment:
why not just use getters value?

Reply:
<img width="843" alt="image" src="https://github.com/EnsembleUI/ensemble/assets/36638657/fa457447-3efa-489e-8df0-c099ad085d84">
The thought behind this was when we render any dynamic widget (e.g in chat) we can call a single method `getValue` for all ensemble input widget.